Modules/Components/Elastix/include/selxMonolithicTransformixComponent.h

```diff
@@ -74,9 +76,10 @@ class MonolithicTransformixComponent :
   typedef typename ComponentBase::CriterionType CriterionType;
   typedef TPixel                                PixelType;
 
-  // fixed and moving image types are all the same, these aliases can be used to be explicit.
-  typedef typename itkImageMovingInterface< Dimensionality, TPixel >::ItkImageType MovingImageType;
-  typedef typename itkImageInterface< Dimensionality, TPixel >::ItkImageType       ResultImageType;
+  // fixed and moving image types are all the same, these aliases can be used to be explicit. 
+  using MovingImageType = typename itkImageMovingInterface< Dimensionality, TPixel >::ItkImageType;
+  using ResultImageType = typename itkImageInterface< Dimensionality, TPixel >::ItkImageType;
+  using DisplacementFieldImageType = typename DisplacementFieldItkImageSourceInterface< Dimensionality, TPixel >::ItkImageType;
```

Comment:
Think of rules how to construct a name (what about ItkImageDisplacementFieldSourceInterface?)